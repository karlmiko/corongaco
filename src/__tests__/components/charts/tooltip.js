import React from 'react'
import renderer from 'react-test-renderer'
import Tooltip from '~components/charts/tooltip'

describe('Components : Charts : Tooltip', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Tooltip top={12} left={30}>
          Test content
        </Tooltip>,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()

    const treeNegativeLeft = renderer
      .create(
        <Tooltip top={12} left={-30}>
          Test content
        </Tooltip>,
      )
      .toJSON()
    expect(treeNegativeLeft).toMatchSnapshot()
  })
})
