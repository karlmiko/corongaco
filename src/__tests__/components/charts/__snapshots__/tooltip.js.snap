// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Charts : Tooltip renders correctly 1`] = `
<div
  className="root"
  style={
    Object {
      "left": 30,
      "top": 12,
    }
  }
>
  Test content
</div>
`;

exports[`Components : Charts : Tooltip renders correctly 2`] = `
<div
  className="root"
  style={
    Object {
      "left": 0,
      "top": 12,
    }
  }
>
  Test content
</div>
`;
